objects/src/notes/mod.rs

```diff
@@ -108,10 +96,11 @@ impl Note {
         serial_num: Word,
         sender: AccountId,
         tag: Felt,
+        note_type: NoteType,
```

Comment:
nit: I would move `note_type` parameter to be before the `tag` parameter.

Reply:
moved